lib/utils/ember.js

```diff
@@ -143,19 +144,28 @@ function isReopenObject(node) {
 }
 
 function isEmberComponent(node, filePath) {
-  return isEmberCoreModule(node, 'Component', filePath);
+  return (
+    types.isClassDeclarationWithSuperClass(node, 'Component') ||
```

Comment:
Added this so that this function would work with native classes. Not sure if this is a good enough way to check that a node is a `Component`, `Controller`, etc. For example, would we need to check the import path for the parent class, since people could rename the import etc? This might be good enough for now, as it will cover most cases. 

We should probably add a similar check to all of the other modules (eg `Service`). If we agree that this is a good enough way to check, I can move the new functions into another PR, add all of the checks in that one and rebase this change off it.

Reply:
Adding this check to all the types of modules sounds good to me. Including an import check would be ideal, perhaps using the new `getSourceModuleNameForIdentifier` util that @alexlafroscia is adding in #548.